pkg/minikube/assets/addons.go

```diff
@@ -593,6 +593,19 @@ var Addons = map[string]*Addon{
 		"GCPAuthWebhook":     "gcr.io",
 		"KubeWebhookCertgen": "registry.k8s.io",
 	}),
+	"volcano": NewAddon([]*BinAsset{
+		MustBinAsset(addons.VolcanoAssets,
+			"volcano/volcano-development.yaml.tmpl",
+			vmpath.GuestAddonsDir,
+			"volcano-deployment.yaml",
+			"0640"),
+	}, false, "volcano", "third-party (volcano)", "", "", map[string]string{
```

Comment:
```suggestion
	}, false, "volcano", "third-party (volcano)", "<github_username>", "", map[string]string{
```

Before adding new addons we would like to have a maintainer listed (GitHub username), this is someone we can reach out to if say the addon is going to break with a new version of Kubernetes or a critical CVE is found in the image as we have little knowledge about the addon itself. Could you please find someone from the project that's willing to take this responsibility and put their GitHub username in the above show spot. Us reaching out will likely be extremely rare so it wouldn't be much or a burden, thanks.

Reply:
I can be the maintainer of this, I am one of the approver of volcano